### PR TITLE
ADD keyboard vortex/M0110_700_iso

### PR DESCRIPTION
**Pull request by gskygithub**
## Description

Add m0110_700 keyboard

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by dunk2k — CHANGES_REQUESTED** [review]

File: keyboards/vortex/m0110/iso/m0110_700/info.json [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+                { "label": "/", "matrix": [8, 5], "x": 11.25, "y": 3 },
+                { "label": "Shift", "matrix": [9, 1], "w": 1.25, "x": 12.25, "y": 3 },
+                { "label": "FN", "matrix": [9, 2], "x": 14, "y": 4 },
+                { "label": "Ctrl", "matrix": [0, 6], "w": 1.25, "x": 0, "y": 4 },
+                { "label": "Win", "matrix": [9, 0], "w": 1.25, "x": 1.25, "y": 4 },
+                { "label": "Alt", "matrix": [9, 3], "w": 1.25, "x": 2.5, "y": 4 },
+                { "label": "Space", "matrix": [9, 4], "w": 6.25, "x": 3.75, "y": 4 },
+                { "label": "RAlt", "matrix": [9, 5], "x": 10, "y": 4 },
+                { "label": "RWin", "matrix": [8, 4], "x": 11, "y": 4 },
+                { "label": "RCtrl", "matrix": [0, 4], "x": 12, "y": 4 }
```
[the remark this answers is not in the document]
Comment (gskygithub) [reply keyboards/vortex/m0110/iso/m0110_700/info.json]: @dunk2k
1. It's a two in one keyboard, let user to customize whether two keys in 1st row or 2 keys.

**Review by dunk2k — CHANGES_REQUESTED** [review]

**Comment by dunk2k** [comment]
@gskygithub 
thanks for the board/PCB images - it appears this board supports split backspace, split right shift, and has support for standard and tsangan bottom rows. Is there a matrix diagram available for this PCB?

File: keyboards/vortex/m0110/iso/m0110_700_iso/info.json [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    "usb": {
+        "device_version": "0.0.2",
+        "pid": "0xE3A5",
+        "vid": "0x342D"
+    },
+    "eeprom": {
+        "driver": "wear_leveling"
+    },
+    "layouts": {
+        "LAYOUT": {
```
Comment (dunk2k) [inline keyboards/vortex/m0110/iso/m0110_700_iso/info.json:40]:
```suggestion
    },
    "layouts": {
        "LAYOUT_60_iso_tsangan_split_bs_rshift": {
```

File: keyboards/vortex/m0110/iso/m0110_700_iso/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  /* Keymap _BL: Base Layer (Default Layer)
+   */
+[0] = LAYOUT(
+  KC_ESC,   KC_1,     KC_2,     KC_3,     KC_4,     KC_5,     KC_6,     KC_7,     KC_8,     KC_9,     KC_0,       KC_MINS,  KC_EQL,   KC_BSLS, KC_BSPC, KC_DEL,
+  KC_TAB,   KC_Q,     KC_W,     KC_E,     KC_R,     KC_T,     KC_Y,     KC_U,     KC_I,     KC_O,     KC_P,       KC_LBRC,  KC_RBRC,  KC_ENT,
+  KC_CAPS,  KC_A,     KC_S,     KC_D,     KC_F,     KC_G,     KC_H,     KC_J,     KC_K,     KC_L,     KC_SCLN,    KC_QUOT,  KC_NUHS,  
+  KC_LSFT,  KC_NUBS,  KC_Z,     KC_X,     KC_C,     KC_V,     KC_B,     KC_N,     KC_M,     KC_COMM,  KC_DOT,     KC_SLSH,  KC_RSFT,  MO(1),
+  KC_LCTL,  KC_LGUI,  KC_LALT,                      KC_SPC,                       KC_RALT,  KC_RGUI,  KC_RCTL ),
+
+[1] = LAYOUT(
```
Comment (dunk2k) [inline keyboards/vortex/m0110/iso/m0110_700_iso/keymaps/default/keymap.c:32]:
```suggestion
[0] = LAYOUT_60_iso_tsangan_split_bs_rshift(
  KC_ESC,   KC_1,     KC_2,     KC_3,     KC_4,     KC_5,     KC_6,     KC_7,     KC_8,     KC_9,     KC_0,       KC_MINS,  KC_EQL,   KC_BSLS, KC_BSPC, KC_DEL,
  KC_TAB,   KC_Q,     KC_W,     KC_E,     KC_R,     KC_T,     KC_Y,     KC_U,     KC_I,     KC_O,     KC_P,       KC_LBRC,  KC_RBRC,  KC_ENT,
  KC_CAPS,  KC_A,     KC_S,     KC_D,     KC_F,     KC_G,     KC_H,     KC_J,     KC_K,     KC_L,     KC_SCLN,    KC_QUOT,  KC_NUHS,  
  KC_LSFT,  KC_NUBS,  KC_Z,     KC_X,     KC_C,     KC_V,     KC_B,     KC_N,     KC_M,     KC_COMM,  KC_DOT,     KC_SLSH,  KC_RSFT,  MO(1),
  KC_LCTL,  KC_LGUI,  KC_LALT,                      KC_SPC,                       KC_RALT,  KC_RGUI,  KC_RCTL ),

[1] = LAYOUT_60_iso_tsangan_split_bs_rshift(
```

File: keyboards/vortex/m0110/iso/m0110_700_iso/keymaps/via/keymap.c [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+  /* Keymap _BL: Base Layer (Default Layer)
+   */
+[0] = LAYOUT(
+  KC_ESC,   KC_1,     KC_2,     KC_3,     KC_4,     KC_5,     KC_6,     KC_7,     KC_8,     KC_9,     KC_0,       KC_MINS,  KC_EQL,   KC_BSLS, KC_BSPC, KC_DEL,
+  KC_TAB,   KC_Q,     KC_W,     KC_E,     KC_R,     KC_T,     KC_Y,     KC_U,     KC_I,     KC_O,     KC_P,       KC_LBRC,  KC_RBRC,  KC_ENT,
+  KC_CAPS,  KC_A,     KC_S,     KC_D,     KC_F,     KC_G,     KC_H,     KC_J,     KC_K,     KC_L,     KC_SCLN,    KC_QUOT,  KC_NUHS,  
+  KC_LSFT,  KC_NUBS,  KC_Z,     KC_X,     KC_C,     KC_V,     KC_B,     KC_N,     KC_M,     KC_COMM,  KC_DOT,     KC_SLSH,  KC_RSFT,  MO(1),
+  KC_LCTL,  KC_LGUI,  KC_LALT,                      KC_SPC,                       KC_RALT,  KC_RGUI,  KC_RCTL ),
+
+[1] = LAYOUT(
```
Comment (dunk2k) [inline keyboards/vortex/m0110/iso/m0110_700_iso/keymaps/via/keymap.c:32]:
```suggestion
[0] = LAYOUT_60_iso_tsangan_split_bs_rshift(
  KC_ESC,   KC_1,     KC_2,     KC_3,     KC_4,     KC_5,     KC_6,     KC_7,     KC_8,     KC_9,     KC_0,       KC_MINS,  KC_EQL,   KC_BSLS, KC_BSPC, KC_DEL,
  KC_TAB,   KC_Q,     KC_W,     KC_E,     KC_R,     KC_T,     KC_Y,     KC_U,     KC_I,     KC_O,     KC_P,       KC_LBRC,  KC_RBRC,  KC_ENT,
  KC_CAPS,  KC_A,     KC_S,     KC_D,     KC_F,     KC_G,     KC_H,     KC_J,     KC_K,     KC_L,     KC_SCLN,    KC_QUOT,  KC_NUHS,  
  KC_LSFT,  KC_NUBS,  KC_Z,     KC_X,     KC_C,     KC_V,     KC_B,     KC_N,     KC_M,     KC_COMM,  KC_DOT,     KC_SLSH,  KC_RSFT,  MO(1),
  KC_LCTL,  KC_LGUI,  KC_LALT,                      KC_SPC,                       KC_RALT,  KC_RGUI,  KC_RCTL ),

[1] = LAYOUT_60_iso_tsangan_split_bs_rshift(
```

**Review by dunk2k — CHANGES_REQUESTED** [review]
including these suggestions could you add the following QMK Community layouts, in `layouts`:
- 60_iso
- 60_iso_split_bs_rshift
- 60_iso_tsangan

**Pull request closed without merging**